package.json

```diff
@@ -218,7 +218,7 @@
         },
         "configurationSnippets": [
           {
-            "label": "Debug a Ruby program",
+            "label": "Ruby: Debug program",
```

Comment:
I changed those to follow the conventions of other languages:

<img width="451" alt="Screenshot 2023-06-28 at 5 18 47 PM" src="https://github.com/Shopify/vscode-ruby-lsp/assets/47848/e4312cd0-c077-4468-baa4-2ec4ce56b98f">